docs/Backend/01. backend-convention.md

```
@@ -0,0 +1,47 @@
+# 백엔드 전체 컨벤션
+
+이 문서는 백엔드 전 도메인에 공통으로 적용하는 기본 컨벤션이다.
+
+## 1) 네이밍 규칙
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)